jupyterhub_traefik_proxy/toml.py

```diff
@@ -64,9 +64,7 @@ def __init__(self, **kwargs):
         except IOError:
             self.log.exception("Couldn't set up traefik's static config.")
         except:
-            self.log.error(
-                "Couldn't set up traefik's static config. Unexpected error:",
-            )
+            self.log.error("Couldn't set up traefik's static config. Unexpected error:")
```

Comment:
add a `raise` here because we don't want to continue if we can't create the configuration.